Dockerfile.glibc

```diff
@@ -39,11 +40,7 @@ ENV  UID=1001
 ENV  GID=1001
 
 RUN  set -ex && \
-     apk --update add libstdc++ curl ca-certificates gcompat && \
```

Comment:
Compile statically so we no longer need this.

Reply:
There are some [gotchas with statically linking glibc](https://stackoverflow.com/questions/57476533/why-is-statically-linking-glibc-discouraged), but I guess iroha might happen to use the APIs that rely on the dynamically linked NSS services, so it works? Should be careful with this stuff

Reply:
do we have to statically link glibc?

Reply:
Good catch, it’s not required for parca i just faced issues with building iroha non statically and use it with alpine image. I will try to use different glibc based container for both stages to make sure that glibc versions are compatible.